src/random.c

```diff
@@ -144,10 +144,10 @@ random_get_bytes (Random    *random,
     }
     return i;
 }
+
 /*
- * Get 64 bits of random data from the parameter 'random' object. If
- * we successfully fill in the supplied 'dest' parameter with random
- * data we return 0. On error return -1.
+ * Get 64 bits of random data from the parameter 'random' object.
+ * On error, return 0 instead of random data.
```

Comment:
This change isn't mentioned anywhere in the commit message but it doesn't look related to new CFLAGS you've added.

Reply:
OK. I found the problem by adding the CCFLAG. I'll add a note to the commit message.